torch/masked/maskedtensor/core.py

```diff
@@ -383,21 +262,10 @@ def __repr__(self):
     def __torch_function__(cls, func, types, args=(), kwargs=None):
         kwargs = kwargs or {}
 
-        from .reductions import _apply_reduction, _is_reduction
-        if _is_reduction(func):
-            return _apply_reduction(func, *args, **kwargs)
-
-        if func in [torch.Tensor.where, torch.where]:
-            _check_args_kwargs_length(args, kwargs, "__torch_function__, torch.where", len_args=3, len_kwargs=0)
-            return _MaskedWhere.apply(*args)
-        if func is torch.Tensor.contiguous:
-            return _MaskedContiguous.apply(args[0])
-        if func is torch.Tensor.to_dense:
-            return _MaskedToDense.apply(args[0])
-        if func is torch.Tensor.to_sparse:
-            return _MaskedToSparse.apply(args[0])
-        if func is torch.Tensor.to_sparse_csr:
-            return _MaskedToSparseCsr.apply(args[0])
+        from ._ops_refs import _MASKEDTENSOR_FUNCTION_TABLE
```

Comment:
I know I'm being a broken record about this, but: it's always better to have imports at the top of a module, unless you're in some rare situation where a) import is very heavy and b) the (single) function that needs it might never be called. see e.g. https://stackoverflow.com/questions/9614051/import-at-module-level-or-at-function-level

Reply:
No you're good! This one was necessary because `._ops_refs` imports from `core.py`, so this was necessary because of circular imports

Reply:
ah got it, thanks!